doc/api/fs.md

```diff
@@ -3815,10 +3815,10 @@ to be notified of filesystem changes.
 * On Aix systems, this feature depends on [`AHAFS`][], which must be enabled.
 
 If the underlying functionality is not available for some reason, then
-`fs.watch` will not be able to function. For example, watching files or
-directories can be unreliable, and in some cases impossible, on network file
-systems (NFS, SMB, etc), or host file systems when using virtualization software
-such as Vagrant, Docker, etc.
+`fs.watch` will not be able to function (an exception may be thrown).
```

Comment:
Which is it? It will not function and will throw an exception? Or it may not function and may throw an exception? Either way, 

```suggestion
`fs.watch` will not be able to function and may throw an exception.
```